support/notulen-queries.js

```diff
@@ -187,14 +187,13 @@ async function fetchParticipationList(zittingUri, bestuursorgaan) {
     ${prefixMap.get("foaf").toSparqlString()}
     ${prefixMap.get("persoon").toSparqlString()}
     SELECT DISTINCT * WHERE {
-        ${sparqlEscapeUri(bestuursorgaan)} org:hasPost ?roleUri.
+      ${sparqlEscapeUri(zittingUri)} besluit:heeftAfwezigeBijStart ?mandatarisUri.
         ?mandatarisUri mandaat:isBestuurlijkeAliasVan ?personUri.
         ?mandatarisUri org:holds ?roleUri.
         ?roleUri org:role ?bestuursfunctieCodeUri.
         ?bestuursfunctieCodeUri skos:prefLabel ?role.
         ?personUri foaf:familyName ?familyName.
         ?personUri persoon:gebruikteVoornaam ?name.
-        filter not exists { ${sparqlEscapeUri(zittingUri)} besluit:heeftAanwezigeBijStart ?mandatarisUri. }
```

Comment:
I don't really understand this, why did we have this line in here?
There's a high probability that I wrote it, but I don't remember to be honest

Reply:
well before we didnt have afwezigeBijStart
so to get the absentees we had to make a query that filters mandatees that are NOT present, rather than just querying for the absentees as we can do now